site/content/docs/alerts/10028.md

```diff
@@ -6,12 +6,16 @@ alerttype: "Passive"
 alertcount: 1
 status: release
 type: alert
+risk: High
 solution: "To avoid the open redirect vulnerability, parameters of the application script/program must be validated before sending 302 HTTP code (redirect) to the client browser. Implement safe redirect functionality that only redirects to relative URI's, or a list of trusted domains"
 references:
    - https://cheatsheetseries.owasp.org/cheatsheets/Unvalidated_Redirects_and_Forwards_Cheat_Sheet.html
    - https://cwe.mitre.org/data/definitions/601.html
-other: ""
+other: "The 301 or 302 response to a request for the following URL appeared to contain user input in the location header:  https://example.com/transact  The user input found was:  place=evil.com  The context was:  http://evil.com"
```

Comment:
Do we care that "\n" new lines are being stripped?

Reply:
Does not look to be a problem.

Reply:
Seems fine in the site.